geode-core/src/main/java/org/apache/geode/internal/security/IntegratedSecurityService.java

```diff
@@ -168,6 +167,19 @@ public void logout() {
     ThreadContext.remove();
   }
 
+  public void logoutJmxUser(String user, Subject currentUser) {
+    try {
+      logger.debug("Logging out " + currentUser.getPrincipal());
```

Comment:
Please wrap this call within a `if (logger.isDebugEnabled()) {}` block.

Reply:
The reason we use `if (logger.isDebugEnabled()) {` is because we wrap log4j's `Logger` inside a Geode `FastLogger` which hits a single volatile flag for `isDebugEnabled()` -- this is a lot faster than what log4j is doing inside `.debug` to check if the log level is `debug` or lower.

Please use log4j parameters instead of string concatenation too:
```
logger.debug("Logging out {}", currentUser.getPrincipal());
```

Reply:
Thanks for comments. Do we need to do same for login and logout?